lib/src/Pos/Diffusion/Full/Block.hs

```diff
@@ -84,13 +86,13 @@ instance Exception BlockNetLogicException where
 -- | Expects sending message to exactly one node. Receives result or
 -- fails if no result was obtained (no nodes available, timeout, etc).
 enqueueMsgSingle ::
-       ( MonadThrow m )
-    => (t2 -> (t1 -> t -> NonEmpty x) -> m (Map NodeId (m b)))
+       ( MonadThrow m, MonadIO m )
```

Comment:
This is a silly thing. I'd like to make it monomorphic (just use `IO`) but that will be quite a bit of extra work. Will do soon but not now.

Reply:
Never mind that, I got rid of the `MonadThrow m` and `liftIO` the `throwIO`s instead.